### PR TITLE
Update intercom app 

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
## Done

- [x] Update Intercom id

## Waiting for

- [x] informing people that should know about this change
- [x] moving all website intercom stuff to the new intercom app
- [ ] other setup stuff for intercom (integrations)

### CODE REVIEW

**Comment by fatboypunk** [comment]
So we're waiting for:
- [ ] informing people that should know about this change
- [ ] moving all website intercom stuff to the new intercom app

right?

**Comment by RonaldDijkstra** [comment]
Correcti

**Comment by RonaldDijkstra** [comment]
Informed the people and transferred a message to the new app.

**Comment by RonaldDijkstra** [comment]
Resolved the conflicts.

**Comment by RonaldDijkstra** [comment]
Ready to go @fatboypunk

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -53,8 +53,33 @@
     <%= partial "partials/popup-ebook" if !is_blog? && current_page.data.popup != false && data.downloads.ebook[I18n.locale] %>
     <%= javascript_include_tag "jquery", "all" %>
     <%= javascript_include_tag "search" if current_page.locale_root_path =~ /zoeken.html/ %>
+<<<<<<< HEAD
```
Comment (fatboypunk) [inline source/layouts/layout.erb]: meh

**Pull request merged**